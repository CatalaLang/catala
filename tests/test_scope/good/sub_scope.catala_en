## Article

```catala
declaration scope A:
  context a content integer
  context b content boolean
  context a_base content integer

declaration scope B:
  context a content integer
  context b content boolean
  context scopeA scope A
  context scopeAbis scope A

scope A:
  definition a_base equals 1
  definition a equals -1
  definition b equals a > 0

scope B:
  definition a equals 42
  definition b equals scopeA.b
  definition scopeA.a under condition a > 0 consequence equals scopeAbis.a_base
```
