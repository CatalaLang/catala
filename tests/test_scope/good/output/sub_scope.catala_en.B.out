[RESULT] Computation successful! Results:
[RESULT] a = 42
[RESULT] b = true
